syntax = "proto3";
package ecomerce;

import "google/protobuf/wrappers.proto";

message ProductID {
    string value = 1;
}

message Product {
    string id = 1;
    string name = 2;
    string description = 3;
    float price = 4;
}

service ProductInfo {
    rpc addProduct(Product) returns (ProductID);
    rpc getProduct(ProductID) returns (Product);
}